### ISSUE TITLE
Does not work in a same remote?

### ISSUE THREAD

**Issue by fu123456** [stsquad/dired-rsync#24]
In Emacs, I open two window (namely two folders in a same remote), and want to rsync a dir to other dir. Then I run 'dired-rsync', but output error as follows:

Warning: Permanently added '[localhost]:50000' (ECDSA) to the list of known hosts.

          8,616 100%    0.00kB/s    0:00:00  
          8,616 100%    0.00kB/s    0:00:00 (xfr#1, to-chk=0/1)
rsync: write failed on "/home/gf/gftemp/notes.tex": Disk quota exceeded (122)
rsync error: error in file IO (code 11) at receiver.c(393) [receiver=3.1.2]

but in the terminal connected by ssh, if I in it run 'rsync -avz [a dir]/notes.tex [an other dir]/', it does work. So why it does not work in my Emacs. Thanks.

**Comment by stsquad** [comment]
Yes currently dired-rsync blindly tries to set up a portfwd to backchannel the rsync. However we should be able to special case the same remote to same remote case.

**Comment by arstum** [comment]
How to implement remote to remote
